README.md

```diff
@@ -73,79 +49,29 @@ I can see whizzy animations in the app
 To make my news reading more fun
 ```
 
-## Mockups
-
-### Headlines page
-
-![Headlines page mockup](/images/news-summary-project-headlines-page-mockup.png)
-
-### Article summary page
-
-![Article page mockup](/images/news-summary-project-article-page-mockup.png)
-
-## API
-
-### API authentication
-
-So that this project can focus on the front-end, we've provided an API that you can use to talk to the Guardian API and the Aylien text summarisation API.  This API's only job is to take your request and add an API key.  This way, you don't have to store API keys in your front-end app.
-
-> Why is it bad to store API keys in your front-end?  If we hadn't provided this API for you to use, how would you avoid this?
-
-### API request rate limits and stubbing
-
-The Guardian and Aylien text summarisation APIs are severely rate-limited.
+### MVC Diagram
 
-**Please stub your tests so we don't exceed the daily limit.  Otherwise, all requests will be rejected and everyone's apps will stop working!**
+# ![MVC Diagram](https://github.com/sabrinalord/news-summary-challenge/blob/master/images/News_MVC_Diagram.png)
```

Comment:
Great planning. I really need to do more of this lol.

Reply:
(Another comment after reading your code) So I guess for the summaries you have planned it but not implemented it yet? Couldn't find that code!